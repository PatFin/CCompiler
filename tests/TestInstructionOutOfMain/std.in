void main ( void )
{

}

putchar(0x42);