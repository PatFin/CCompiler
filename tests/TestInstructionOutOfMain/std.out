Compilation failed due to syntax errors : exiting with return value 1
